Src/Support/IntegrationTests/ExternalAccountCredentialTests.cs

```diff
@@ -109,7 +109,8 @@ public async Task AwsCredentials()
             // A final sanity-check.
             Assert.NotNull(buckets.Items);
 
-            string AwsCredentialValue(string tagName) => awsCredentialXml.GetElementsByTagName(tagName)[0].InnerText;
+            string AwsCredentialValue(string tagName) =>
+                awsCredentialXml.Descendants().Where(element => element.Name.LocalName == tagName).Single().Value;
```

Comment:
Nit: you can change `.Where(xxx).Single()` to `.Single(xxx)` to simplify it a bit.

Reply:
Ah true, will change and rebase all in a couple of commits.